README.md

```diff
@@ -58,15 +58,15 @@ const endpointConfig = {
         method: 'POST',
         afterSuccess: ({ dispatch, request, getState }) => {
             // After successful post, invalidate the cache of the getArticle call, so it gets re-triggered.
-            dispatch(invalidateApiDataRequest('getArticle', {articleId: request.params.articleId, userId: getState().userId})); 
+            dispatch(invalidateRequest('getArticle', {articleId: request.params.articleId, userId: getState().userId})); 
         }
     }
 };
 
 // Configure store and dispatch config before you render components
 
 const store = createStore(combineReducers({apiData: reducer}), applyMiddleware(thunk));
-store.dispatch(configureApiData({}, endpointConfig));
+store.dispatch(configure({}, endpointConfig));
 ```
 
 
```

Comment:
helemaal onderin staat nog een configure die niet aangepast is (om de een of andere reden mag ik deze comment niet daar zetten...)